lib/wise_homex/json_parser.ex

```diff
@@ -20,6 +20,8 @@ defmodule WiseHomex.JSONParser do
              "message-reports" => WiseHomex.MessageReport,
              "pongs" => WiseHomex.Pong,
              "radiator-import-results" => WiseHomex.RadiatorImportResult,
+             "ready-installation-matches" => WiseHomex.ReadyInstallationMatch,
+             "ready-installation-already-matched" => WiseHomex.ReadyInstallationMatch,
```

Comment:
Is this correct? The link endpoints return a `"ready-installations"`

Reply:
I think it is right. If you look in `V6.ReadyInstallationMatchController`.

Reply:
Ah. But the JSONParser is only for return values of the API. I think this is only an input type, right? The output type is `ready-installation-matches`. But the two link endpoints return something of type `ready-installations`. That should also be included here, right?

Reply:
@lasseebert is correct :heavy_check_mark: